### PR TITLE
Profile Picture on Profile Page

### PR DESCRIPTION
**Pull request by hbrock**
I change the logic for show the Gravatar in Profiles.cshtml.cs

### CODE REVIEW

File: src/NuGetGallery/Views/Users/Profiles.cshtml [hunk 1]
```diff
@@ -6,9 +7,13 @@
 
 
 @section SideColumn {
-    @if (String.IsNullOrEmpty(Model.EmailAddress))
+    @* Change to not null or empty *@
```
Comment (maartenba) [inline src/NuGetGallery/Views/Users/Profiles.cshtml]: What is meant by this comment? Remove it if not needed.

**Comment by maartenba** [comment]
Can you limit the PR to include just the `src/NuGetGallery/Views/Users/Profiles.cshtml` file that was updated?

**Comment by hbrock** [comment]
Is there away I can limit it to just that one file for github?

**Comment by maartenba** [comment]
That will need some git-fu. Here's what you could do:
* revert your commits locally (keep a copy of the files you modified so you don't lose changes) - git revert HEAD~2 --hard will do
* copy your changes back
* do a selective commit of your changes that has *only* that file
* git push --force (that's not recommended practice but in this case will do the trick)

**Comment by hbrock** [comment]
The changes should be fixed...If I did not do anything wrong.

**Comment by hbrock** [comment]
I seen I had an error with the last update. It should be working now.

**Pull request merged**